src/test/e2e/tests/specs.test.ts

```diff
@@ -156,7 +146,7 @@ describe('Unverified user', () => {
     it('should select \'Single Justice Procedure case\' option and navigate to Single Justice Procedure case page', async () => {
       await viewOptionPage.selectOption('SingleJusticeProcedureRadioButton');
       singleJusticeProcedurePage = await viewOptionPage.clickContinueSingleJusticeProcedure();
-      expect(await singleJusticeProcedurePage.getPageTitle()).toEqual('Single Justice Procedure cases');
+      expect(await singleJusticeProcedurePage.getPageTitle()).toEqual('Summary of Publications for Single Justice Procedure (SJP)');
```

Comment:
same here, this is the step before the SJP page right? theres a test that sits before this one to get to the page that the test was origionally talking about